src/sentry/models/releasefile.py

```diff
@@ -75,24 +81,29 @@ class ReleaseFile(Model):
     public_objects = PublicReleaseFileManager()
 
     class Meta:
-        unique_together = (("release", "ident"),)
-        index_together = (("release", "name"),)
+        unique_together = (("release_id", "ident"),)
+        index_together = (("release_id", "name"),)
         app_label = "sentry"
         db_table = "sentry_releasefile"
 
     def save(self, *args, **kwargs):
+        from sentry.models import Distribution
+
         if not self.ident and self.name:
-            dist = self.dist_id and self.dist.name or None
+            dist = None
+            if self.dist_id:
+                dist = Distribution.objects.get(pk=self.dist_id).name
             self.ident = type(self).get_ident(self.name, dist)
         return super().save(*args, **kwargs)
 
     def update(self, *args, **kwargs):
         # If our name is changing, we must also change the ident
         if "name" in kwargs and "ident" not in kwargs:
-            dist = kwargs.get("dist") or self.dist
-            kwargs["ident"] = self.ident = type(self).get_ident(
-                kwargs["name"], dist and dist.name or dist
-            )
+            dist_name = None
+            dist_id = kwargs.get("dist_id") or self.dist_id
+            if dist_id:
+                dist_name = Distribution.objects.only("name").get(pk=dist_id).name
```

Comment:
Our codebase explicitly breaks usage of `.only` (https://github.com/getsentry/sentry/blob/a438f43cd0ad85fbf6a43a25726e9d32318ad697/src/sentry/db/models/manager.py#L65). 

You can use `values_list` instead:
`dist_name = Distribution.objects.filter(pk=dist_id).values_list("name", flat=True)[0]`

Reply:
Changed to `values_list`.